aws/rust-runtime/aws-config/src/profile/retry_config.rs

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Load retry configuration properties from an AWS profile
+
+use crate::provider_config::ProviderConfig;
+use aws_types::os_shim_internal::{Env, Fs};
+use smithy_types::retry::{RetryConfig, RetryMode};
+use std::str::FromStr;
+
+/// Load retry configuration properties from a profile file
+///
+/// This provider will attempt to load AWS shared configuration, then read retry configuration properties
+/// from the active profile.
+///
+/// # Examples
+///
+/// **Loads 2 as the `max_attempts` to make when sending a request
+/// ```ini
+/// [default]
+/// max_attempts = 2
+/// ```
+///
+/// **Loads `standard` as the `retry_mode` _if and only if_ the `AWS_PROFILE` environment variable is set
```

Comment:
probably should be: the `other` profile is selected

There are other ways to override the profile